src/app/app.module.ts

```diff
@@ -1,10 +1,22 @@
 import { BrowserModule } from '@angular/platform-browser';
 import { NgModule } from '@angular/core';
+import { RouterModule, Routes } from '@angular/router';
 import { FormsModule } from '@angular/forms';
 import { HttpModule } from '@angular/http';
 import { AngularFireModule } from 'angularfire2';
 
 import { AppComponent } from './app.component';
+import { HeaderComponent } from './header/header.component';
+import { ProfileComponent } from './profile/profile.component';
+import { CommunityComponent } from './community/community.component';
+import { DashboardComponent } from './dashboard/dashboard.component';
```

Comment:
Each of these will eventually be pages and we will want to lazy load them so that people don't get ALL the pages up front and we can serve them the necessary code as they require it. For now just make them each a module with their own router and we will figure out how to lazy load